src/index.html

```diff
@@ -6,9 +6,69 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
+    <header>
+      <a href="#home" class="logo">
+        <img src="images/logo.png"
+             alt="moyo"
+             class="logo__image">
```

Comment:
```suggestion
        <img 
          src="images/logo.png"
          alt="moyo"
          class="logo__image"
        >
```
Consider following the code style guide